codegen-server-test/build.gradle.kts

```diff
@@ -36,7 +36,8 @@ val allCodegenTests = listOf(
     CodegenTest("com.amazonaws.simple#SimpleService", "simple"),
     CodegenTest("aws.protocoltests.restjson#RestJson", "rest_json"),
     CodegenTest("com.amazonaws.ebs#Ebs", "ebs"),
-    CodegenTest("com.amazonaws.s3#AmazonS3", "s3")
+    CodegenTest("com.amazonaws.s3#AmazonS3", "s3"),
+    CodegenTest("com.aws.example#PokemonService", "pokemon-service-sdk")
```

Comment:
Shall we use underscores here?

I think that crates.io allowing dashes in crate names was a mistake, because they are not valid identifiers.

Reply:
Yeah, good call out. I will fix this.